### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by atodorov**

### CODE REVIEW

**Comment by atodorov** [comment]
@clumens can you please review. I have more coming but let's keep the PRs as small as possible.

**Comment by clumens** [comment]
Exactly how many more sets of test PRs are you planning on creating?

**Comment by atodorov** [comment]
> Exactly how many more sets of test PRs are you planning on creating?

Well I don't really know. I'm going over the files one by one and adding tests as required. Until a PR is merged or I take some time to do something else I usually keep adding new commits to it.  I'm trying to keep changes isolated inside individual commits so they are easier to amend when necessary. 

I will fix the comments above shortly.

File: tests/commands/volgroup.py [hunk 1]
```diff
@@ -108,17 +117,18 @@ def runTest(self):
...
 
         # Pass - correct usage.
-        self.assert_parse("volgroup vg.01 pv.01 --reserved-space=1000",
-                          "volgroup vg.01%s --reserved-space=1000 pv.01\n" % def_pesize_str)
-        self.assert_parse("volgroup vg.01 pv.01 --reserved-percent=50",
-                          "volgroup vg.01%s --reserved-percent=50 pv.01\n" % def_pesize_str)
+        self.assert_parse("volgroup vg.01 pv.01 --reserved-space=1",
+                          "volgroup vg.01%s --reserved-space=1 pv.01\n" % def_pesize_str)
+        self.assert_parse("volgroup vg.01 pv.01 --reserved-percent=1",
+                          "volgroup vg.01%s --reserved-percent=1 pv.01\n" % def_pesize_str)
```
Comment (atodorov) [inline tests/commands/volgroup.py]: This is a boundary value and we really should be testing with boundary values as well. The fact that we weren't was revealed during my mutation testing experiments. In this case it didn't matter which value we were testing with (1 or 50) so I decided to update the value instead of adding a whole new test for it.

**Comment by atodorov** [comment]
hi folks, it's been 9 months since this PR was opened. Can I get a review or merge ?

**Comment by atodorov** [comment]
ping, any updates here ?

**Comment by dcantrell** [comment]
Can you rebase the patch set and resolve the issues discussed?

**Comment by atodorov** [comment]
@dcantrell I've rebased to the latest master. Previous comments have been resolved meanwhile so nothing more to add on my side.  Travis seems to fail with a problem with pykickstart

**Comment by dcantrell** [comment]
Thanks for rebasing and resolving previous issues.  Merging.

**Comment by larskarlitski** [comment]
This broke CI. I'm adding fixes on https://github.com/dcantrell/pykickstart/pull/252
